src/sage/combinat/words/finite_word.py

```diff
@@ -5675,7 +5674,7 @@ def abelian_vectors(self, n):
         size = alphabet.cardinality()
         if size == float('inf'):
             raise TypeError("The alphabet of the parent is infinite; define"
```

Comment:
I don't expect you to do anything about this in this PR because it would be out of scope, but it would be nice to standardize either using `'` or `"` for strings throughout Sage.

Reply:
maybe, but this is an herculean task

Reply:
I think it could be done with auto-formatting, but yeah it would be really big.